packages/syntax-core/src/Button/Button.tsx

```diff
@@ -144,12 +141,16 @@ const Button = forwardRef<HTMLButtonElement, ButtonType>(
           <StartIcon className={classNames(styles.icon, iconSize[size])} />
         )}
         {((loading && loadingText) || (!loading && text)) && (
-          <div className={styles.textContainer}>
```

Comment:
@somethiiing can we remove `textContainer` from the CSS since it's no longer used?